File: streaming/java/Tutorial.java
import org.apache.spark.*;
import org.apache.spark.api.java.*;
import org.apache.spark.api.java.function.*;
import org.apache.spark.streaming.*;
import org.apache.spark.streaming.twitter.*;
import org.apache.spark.streaming.api.java.*;
import twitter4j.*;
import java.util.Arrays;
import scala.Tuple2;

public class Tutorial {
  public static void main(String[] args) throws Exception {

    // Checkpoint directory 
    String checkpointDir = TutorialHelper.getCheckpointDirectory();

    // Configuring Twitter credentials 
    String apiKey = "";
    String apiSecret = "";
    String accessToken = "";
    String accessTokenSecret = "";
    TutorialHelper.configureTwitterCredentials(apiKey, apiSecret, accessToken, accessTokenSecret);

    // Your code goes here

  }
}

File: streaming/Tutorial.scala
import spark._
import spark.streaming._
import SparkContext._
import StreamingContext._
import TutorialHelper._

object Tutorial {
  def main(args: Array[String]) {
    
    // Location of the Spark directory 
    val sparkHome = "/root/spark"
    
    // URL of the Spark cluster
    val sparkUrl = TutorialHelper.getSparkUrl()

    // Location of the required JAR files 
    val jarFile = "target/scala-2.9.2/tutorial_2.9.2-0.1-SNAPSHOT.jar"

    // HDFS directory for checkpointing
    val checkpointDir = TutorialHelper.getHdfsUrl() + "/checkpoint/" 

    // Twitter credentials from login.txt
    val (twitterUsername, twitterPassword) = TutorialHelper.getTwitterCredentials()
   
    // Your code goes here
  }
}

